prestodb/client.py

```diff
@@ -230,6 +233,19 @@ def __init__(
         else:
             # mypy cannot follow module import
             self._http_session = self.http.Session()  # type: ignore
+
+        self.credentials = None
+        self.auth_req = None
+        if service_account_file is not None:
+            self.auth_req = google.auth.transport.requests.Request()
+            self.credentials = service_account.Credentials.from_service_account_file(
+                service_account_file, scopes=[constants.GCS_READ_ONLY])
+            self.credentials.refresh(self.auth_req)
+            token = {
+                constants.PRESTO_EXTRA_CREDENTIAL: constants.GCS_CREDENTIALS_OAUTH_TOKEN_KEY + '=' + self.credentials.token
```

Comment:
nit: It would be better to use formatting string instead of string concatenation.

Reply:
Updated, thanks!